musicbrainz-dockerfile/Dockerfile

```diff
@@ -2,9 +2,8 @@ FROM metabrainz/base-image
 LABEL maintainer=jeffsturgis@gmail.com
 
 ENV DOCKERIZE_VERSION v0.5.0
-RUN wget https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
-    tar -C /usr/local/bin -xzvf dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz && \
-    rm -f dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz
+RUN curl -sSL https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-linux-amd64-$DOCKERIZE_VERSION.tar.gz | \
+    tar -C /usr/local/bin -xzv
```

Comment:
This requires to set the shell option `pipefail` to reliably detect failures, as in https://github.com/metabrainz/musicbrainz-docker/blob/mbvm-38-dev/build/musicbrainz/Dockerfile#L10

Reply:
Good to know, thank you!